### PR TITLE
defining build arg for papermill package (should have done this previ…

### PR DESCRIPTION
**Pull request by jay-bhambhani**
…ously)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -18,6 +18,9 @@ ARG AIRFLOW_DEPS=""
 ARG PYTHON_DEPS=""
 ENV AIRFLOW_HOME=${AIRFLOW_USER_HOME}
 
+# Papermill
+ARG PAPERMILL_PACKAGE=""
```
Comment (devstein) [inline Dockerfile]: default this to "papermill"

**Review by devstein — APPROVED** [review]
one comment

File: Dockerfile (hunk 1 above)
Comment (jay-bhambhani) [reply Dockerfile]: sounds good

Comment (jay-bhambhani) [reply Dockerfile]: @devstein just to check - i did this correctly? im not a moron am i?

**Pull request merged**